### PR TITLE
pr1

### PR DESCRIPTION
**Pull request by dascalubogdan**

### CODE REVIEW

File: WebApplication1.Tests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace WebApplication1.Tests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+            //ssssdddsssdddssddddddddd
```
Comment (dascalubogdan) [inline WebApplication1.Tests/UnitTest1.cs]: change

File: WebApplication1.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace WebApplication1.Tests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
```
Comment (dascalubogdan) [inline WebApplication1.Tests/UnitTest1.cs]: change

Comment (dascalubogdan) [inline WebApplication1.Tests/UnitTest1.cs]: ok

File: WebApplication1/WebApplication1.csproj [hunk 3]
```diff
@@ -386,13 +386,6 @@
       </FlavorProperties>
     </VisualStudio>
   </ProjectExtensions>
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```
Comment (dascalubogdangmail) [inline WebApplication1/WebApplication1.csproj:389]: undo this
@dascalubogdan

**Review by dascalubogdangmail — CHANGES_REQUESTED** [review]
made some comments

File: WebApplication1/Controllers/MeController.cs [hunk 4]
```diff
@@ -40,7 +40,7 @@ private set
             }
         }
 
-        // GET api/Me
+        // GET api/Me sdasdasdasdaasdssssss
```
Comment (dascalubogdangmail) [inline WebApplication1/Controllers/MeController.cs:43]: ?

**Review by dascalubogdangmail — CHANGES_REQUESTED** [review]

File: WebApplication1/WebApplication1.csproj [hunk 5]
```diff
@@ -386,13 +386,6 @@
...
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Microsoft.Net.Compilers.1.0.0\build\Microsoft.Net.Compilers.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.Net.Compilers.1.0.0\build\Microsoft.Net.Compilers.props'))" />
-    <Error Condition="!Exists('..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.0\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.0\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props'))" />
```
Comment (dascalubogdangmail) [inline WebApplication1/WebApplication1.csproj:394]: ggggg

**Review by dascalubogdangmail — CHANGES_REQUESTED** [review]

**Comment by dascalubogdan** [comment]
klose

**Pull request merged**